contrib/zpages/src/main/java/io/opencensus/contrib/zpages/StatszZPageHandler.java

```diff
@@ -131,6 +130,11 @@ public void emitHtml(Map<String, String> queryMap, OutputStream outputStream) {
     out.write("<meta charset=\"utf-8\">\n");
     out.write("<title>StatsZ</title>\n");
     out.write("<link rel=\"shortcut icon\" href=\"//www.opencensus.io/favicon.ico\"/>\n");
+    out.write(
```

Comment:
Please move this part to `emitStyles()`.

Reply:
Consider a separate .css file that we embed into the binary. This way we can share the same file between multiple implementations (languages).

Reply:
Yes that sounds better, though I don't know how difficult it is to apply the same css file to all pages, since each zpage has its own class/id names. @adamgarza what do you think?